controllers/openshift_controller.go

```diff
@@ -1445,13 +1445,13 @@ func (r *KataConfigOpenShiftReconciler) updateNodeLabels() (err error) {
 
 	if err := r.Client.List(context.TODO(), workerNodeList, listOpts...); err != nil {
 		r.Log.Error(err, "Getting list of nodes failed")
-		return err
+		return false, err
```

Comment:
This could be a user for `dummyBool()` as suggested earlier.

Reply:
Not really, I don't think.  The semantics of the `bool` return value is to tell the caller whether any "kata-oc" node-role labels were added or removed (as suggested by the name of the boolean).  When getting the worker list fails then the function is obviously unable to change any labels so it returns `false` to the caller to indicate this.

Reply:
Ok, I get it now : unlike with the other commit, the bool return value is meaningful even if an error is returned.